client/src/app/site/pages/meetings/pages/motions/components/motion-forward-dialog/components/motion-forward-dialog/motion-forward-dialog.component.html

```diff
@@ -132,6 +132,9 @@
                                         <span>{{ data.motion[0].number }}</span>
                                     </span>
                                 }
+                                <mat-checkbox [(ngModel)]="markAmendmentsAsForwarded">
```

Comment:
Are you sure that you want to add this inside the radio button? 
I guess this might be better placed outside the `radio-group`.